api/src/test_k6_rate_limiter.js

```diff
@@ -0,0 +1,58 @@
"Provence-Alpes-Côte d\'Azur","email":"hugo.lecourt${randomNumber}+guagua@selego.co","firstName":"Olga","lastName":"ALTENWERTH","address":"34 Boulevard charles livon","birthCity":"New Fannie","birthCityZip":"43267-8897","city":"Marseille","cityCode":"13055","phone":"0622475529","zip":"13007","parent1Email":"Braden_Pouros@gmail.com","parent1FirstName":"Matteo","parent1LastName":"Kuphal","parent1Phone":"388-257-2920 x255","parent2Email":"Hilbert68@hotmail.com","parent2FirstName":"Alberto","parent2LastName":"Gutmann","parent2Phone":"959-903-7370","parent1Inscription2023Token":"169add62227d3ca607896881bb2640473f5acba3","parent2Inscription2023Token":"cbe2b7f4a69981b30ff5fdb164b9e1ab8d4c09ec","birthdateAt":"2007-08-12T11:00:00.000Z","latestCNIFileExpirationDate":"2025-08-12T11:00:00.000Z","inscriptionDoneDate":"2022-11-30T10:08:43.081Z","parent1ValidationDate":"2022-11-30T10:08:43.081Z","frenchNationality":"true","schooled":"true","schoolName":"Collège Miles de Noyers","schoolType":"CLG","schoolAddress":"24 bis rue du Pont Neuf","schoolZip":"89310","schoolCity":"Noyers","schoolDepartment":"Yonne","schoolRegion":"Bourgogne-Franche-Comté","schoolCountry":"FRANCE","schoolId":"634026949bc0ed3c3291c4fa","grade":"TermGT","inscriptionStep2023":"CONFIRM","addressVerified":"true","allergies":"false","birthCountry":"France","country":"France","employed":"false","gender":"female","handicap":"false","handicapInSameDepartment":"false","isRegionRural":"true","paiBeneficiary":"false","populationDensity":"","ppsBeneficiary":"false","reducedMobilityAccess":"false","situation":"GENERAL_SCHOOL","specificAmenagment":"false","specificAmenagmentType":"","consentment":"true","parent1ContactPreference":"phone","parent1Status":"mother","parent2ContactPreference":"email","parent2Status":"father","CNIFileNotValidOnStart":"false","latestCNIFileCategory":"cniOld","informationAccuracy":"true","parent1DataVerified":"true","parent1AllowImageRights":"true","parent1AllowSNU":"true","parent1OwnAddress":"false","parentAllowSNU":"true","rulesParent1":"true","lastStatusAt":"2023-07-10T07:40:15.615Z","historic":[],"lastLoginAt":"2023-06-26T15:35:16.039Z","missionsInMail":[],"notes":[],"createdAt":"2022-11-30T10:08:43.081Z","updatedAt":"2023-08-08T09:40:56.958Z","__v":7,"withdrawnMessage":"hj","withdrawnReason":"unavailable_perso","parent1PhoneZone":"FRANCE","parent2PhoneZone":"FRANCE","phoneZone":"FRANCE","cohortChangeReason":"Non disponibilité pour motif familial ou personnel","originalCohort":"Février 2023 - C","isTravelingByPlane":"true","lastLogoutAt":"2023-06-28T05:47:05.716Z","lastActivityAt":"2023-08-04T14:02:11.966Z","cohesionCenterId":"63873278a4ec702331abf14b","hasMeetingInformation":"true","sessionPhase1Id":"648862275510530683a03e93","transportInfoGivenByLocal":"false","token2FAExpires":"2023-06-29T07:24:32.947Z","deplacementPhase1Autonomous":"false","ligneId":"64915e9d238d7cad6742fc7f","meetingPointId":"63872e9e1f87fb1fe62dc2e7","cohesionStayPresence":"true","applications":[]}`,
```

Comment:
> I haven't started to look into the tests yet
Do we have a seed to populate the db for test? We might want to fetch the users first than create a function to update a parameter or whatever then send a put request
Also, the hard coded `localhost` will not work in cloud test pipelines

Reply:
It's for running the loadtest locally, i did this test with k6 recorder